<launch>
    <node name="drag_centerline_planner_daniel" pkg="rr_iarrc" type="drag_centerline_planner_daniel" output="screen">
        <!--PID Controller Constants-->
        <param name="PID_kP" type="double" value="0.0003" />
        <param name="PID_kI" type="double" value="0.0" />
        <param name="PID_kD" type="double" value="0.0" />

        <param name="speed" type="double" value="0" />

        <param name="maxTurnLimitRadians" type="double" value="0.44" />

        <param name="subscription_node" type="string" value="/camera_center/image_color_rect/lines/detection_img"/>
    </node>
</launch>
